system/jlib/jfile.cpp

```diff
@@ -2071,6 +2136,25 @@ CFileIO::CFileIO(IFile * _creator, HANDLE handle, IFOmode _openmode, IFSHmode _s
     extraFlags = static_cast<IFEflags>(extraFlags | expertEnableIFileFlagsMask);
     extraFlags = static_cast<IFEflags>(extraFlags & ~expertDisableIFileFlagsMask);
 
+    if (isContainerized() && (openmode!=IFOread)) // only containerized (with planes writing to storage types like blob for now)
+    {
+        const char *dataPath = "/var/lib/HPCCSystems/hpcc-data/";
+        if (startsWith(querySafeFilename(), dataPath))
+        {
+            DBGLOG("CFileIO: file '%s' is in data path '%s'", dataPath, querySafeFilename());
+            // fileSyncMaxRetrySecs applies to IFEsync and IFEsyncAtClose
+            const char *planeName = nullptr; // TBD - deduce plance from file path
```

Comment:
Are you planning to implement this?

Possibly worth implementing an efficient function to do that mapping, in case it is useful elsewhere.